server/src/main/java/com/google/coffeehouse/servlets/CreateClubServlet.java

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.coffeehouse.servlets;
+
+import com.google.coffeehouse.common.Club;
+import com.google.coffeehouse.util.IdentifierGenerator;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import java.io.IOException;
+import java.util.Map;
+import java.util.stream.Collectors;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** 
+ * Servlet to create a {@link Club} from Http Post Request Body (in JSON format),
+ * save it in database, and return it in JSON format.
+ */
+@WebServlet("/create-club")
+public class CreateClubServlet extends HttpServlet {
+
+  /** 
+   * The error string sent by the response object in doPost when the body of the 
+   * post request cannot be used to construct a {@link Club} for any reason.
+   */
+  public static final String BODY_ERROR = "- unable to parse body.";
```

Comment:
What is the `- ` for here?

Reply:
Upon error, this line will be prepended by the string "Error 400 ". The dash is just for formatting (it was chosen instead of a colon because the colon would be placed after the space). 
![image](https://user-images.githubusercontent.com/65669542/86827928-365de680-c058-11ea-858f-9251407a052e.png)